### PR TITLE
Sorry eli

### PR DESCRIPTION
**Pull request by iiAzui**
Added a bunch of stuff, it's mostly self explanatory.

@lishaduck, have fun re-adding everything you did, lol.

### CODE REVIEW

**Comment by iiAzui** [comment]
or just losing what you added

**Comment by iiAzui** [comment]
@lishaduck

**Comment by iiAzui** [comment]
Changes:
Tutorial map added
Main Menu Added
Juices and scores added
some re organization

File: GameOver/GameOver.tscn
Comment (lishaduck) [inline GameOver/GameOver.tscn:1]: snake_case

**Comment by lishaduck** [comment]
Merging will require a rebase. Will do later.

**Comment by lishaduck** [comment]
> Merging will require a rebase. Will do later.

Pushed it partially. Will finish later.

**Comment by lishaduck** [comment]
> > Merging will require a rebase. Will do later.
> 
> Pushed it partially. Will finish later.

Almost finished. I'll have to revert the map changes, but there're still here. Pull main once it's merged and re-add them.

**Comment by lishaduck** [comment]
What's the font? We'll need to include the license.

**Comment by lishaduck** [comment]
Superseded by #28.

**Pull request closed without merging**